content/en/account_management/faq/how-do-i-grant-or-remove-a-global-permission-to-or-from-a-role.md

```diff
@@ -0,0 +1,128 @@
+| monitors_write               | 48ef71ea-d8b1-11e9-a77a-93f408470ad0 | Ability to change, mute, and delete  monitors |
+| monitors_downtime            | 4d87d5f8-d8b1-11e9-a77a-eb9c8350d04f | Ability to set downtimes for your monitors    |
+
+{{% /tab %}}
+{{% tab "Datadog EU site" %}}
+| name                         | UUID                                 | description                                   |
+|------------------------------|--------------------------------------|-----------------------------------------------|
+| admin                        | f1624684-d87d-11e8-acac-efb4dbffab1c | Read and write permission to all of datadog   |
+| standard                     | f1666372-d87d-11e8-acac-6be484ba794a | Read and write permission to most of datadog  |
+| logs_read_index_data         | 4fbb1652-dd15-11e8-9308-77be61fbb2c7 | Read a subset of all log indexes              |
+| logs_modify_indexes          | 4fbd1e66-dd15-11e8-9308-53cb90e4ef1c | Update the definition of log indexes          |
+| logs_live_tail               | 4fbeec96-dd15-11e8-9308-d3aac44f93e5 | Access the live tail feature                  |
+| logs_write_exclusion_filters | 4fc2807c-dd15-11e8-9308-d3bfffb7f039 | Update a subset of the exclusion filters      |
+| logs_write_pipelines         | 4fc43656-dd15-11e8-9308-f3e2bb5e31b4 | Update a subset of the log pipelines          |
+| logs_write_processors        | 505f4538-dd15-11e8-9308-47a4732f715f | Update the log processors in an index         |
+| logs_write_archives          | 505fd138-dd15-11e8-9308-afd2db62791e | Update the external archives configuration    |
+| logs_public_config_api       | bd837a80-6cb2-11e9-8fc4-339b4b012214 | Access the Logs Public Config API (r/w)       |
+| logs_generate_metrics        | 06f715e2-aed9-11e9-aac6-eb5723c0dffc | Access the Generate Metrics feature           |
+| dashboards_read              | 2147a4f0-d3d9-11e9-a614-83d5d3c791ee | Ability to view dashboards                    |
+| dashboards_write             | 2149e512-d3d9-11e9-a614-bb8f0dcf0205 | Ability to create and change dashboards       |
+| dashboards_public_share      | 214c10b2-d3d9-11e9-a614-3759c7ad528f | Ability to share dashboards externally        |
+| monitors_read                | c898551e-d8b2-11e9-a336-e3a79c23bd8d | Ability to view monitors                      |
+| monitors_write               | cdc3e3d2-d8b2-11e9-943b-e70db6c573b8 | Ability to change, mute, and delete  monitors |
+| monitors_downtime            | d3159858-d8b2-11e9-a336-e363d6ef331b | Ability to set downtimes for your monitors    |
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Granting Permissions Within Limited Scopes
+
+Certain permissions can be granted within a limited scope. This can be done manually from the Datadog application in [the Pipelines Page][1], or programmatically via the Role API if the correct "scope" is added in the payload. The following permissions can be granted within a limited scope:
```

Comment:
```suggestion
Certain permissions can be granted within a limited scope. This can be done manually from the Datadog site on the [Pipelines page][1], or programmatically with the Role API if the correct `scope` is added to the payload. The following permissions are available within a limited scope:
```